plugins/modules/cpanm.py

```diff
@@ -56,6 +56,18 @@
       - Only install dependencies.
     type: bool
     default: false
+  install_recommendations:
+    description:
+      - Installs dependencies declared as recommends per META spec.
+      - When these dependencies fail to install, cpanm continues the installation, since they are just recommendation.
```

Comment:
We should be telling the user about the 3 options they have. Maybe something like:
```suggestion
      - If V(true), installs dependencies declared as recommends per META spec.
      - If V(false), it ensures the dependencies declared as recommends are not installed, overriding any decision made earlier in E(PERL_CPANM_OPT).
      - If parameter is not set, C(cpanm) will use its existing defaults.
      - When these dependencies fail to install, cpanm continues the installation, since they are just recommendation.
```

Reply:
Yep, I agree. Explicit is better. I'll just use your suggestion.
By the way what's the keyword for the doc about those doc functions : `V(true)`, `V(false)`, C(cpanm) and E(PERL_CPANM_OPT) ?

Reply:
Found it: https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-within-module-documentation